classes/PodsUI.php

```diff
@@ -3030,7 +3068,18 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 								<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: c26b5d4b2a37afe7b8c98baaf1a2c8d1</sub>